packages/rxtranscribe/demo/runDemo.js

```diff
@@ -6,28 +6,27 @@ const {tap} = require('rxjs/operators');
 const {fromFile} = require('@bottlenose/rxfs');
 
 // const {transcribe} = require('../src/index');
-const {toAWS} = require('../build/index');
-const {toDeepSpeech} = require('../build/index');
-// import {toGCP} from '../src/index';
+const {toAWS, toDeepSpeech, toGCP} = require('../build/index');
 // import {toDeepgram} from '../src/index';
 
 console.log('running demo');
 
+
 function createOperator({
   strategy,
   region = 'us-east-1',
   modelDir,
-  codec = 'wav',
+  sampleRate
 }) {
   switch (strategy) {
     case 'deepspeech':
-      return toDeepSpeech({modelDir, codec: 'pcm'});
+      return toDeepSpeech({modelDir});
```

Comment:
❓ I should double-check this...  Not sure if it's the same as what I have or not...

Reply:
Might have been some incorrect merging that I didn't notice. Good spot.